packages/block-library/src/navigation/editor.scss

```diff
@@ -441,28 +442,28 @@ $color-control-label-height: 20px;
 	}
 }
 
-.has-fixed-toolbar .wp-block-navigation__responsive-container.is-menu-open {
+.has-fixed-toolbar .wp-block-navigation.is-responsive-menu-open {
```

Comment:
I like the new simpler classnames. Hopefully it's just a missing update that's causing the justifications to not work on the frontend, pretty sure there are explicit justification rules that target this one.

Reply:
Did you test the latest changes to this branch? Overlay menu justification wasn't working in the first iteration, but I've fixed it since. If you're testing locally, you might have to delete your local PR branch and check it out again, because I rebased it yesterday.